src/component/variantPage/biologicalFunction/Oncokb.tsx

```diff
@@ -0,0 +1,173 @@
+import * as React from 'react';
+import { observer } from 'mobx-react';
+import classNames from 'classnames';
+
+import BiologicalFunctionStyles from './BiologicalFunction.module.scss';
+import { DefaultTooltip } from 'cbioportal-frontend-commons';
+import functionalGroupsStyle from '../functionalGroups.module.scss';
+import { IndicatorQueryResp } from 'cbioportal-frontend-commons/api/generated/OncoKbAPI';
+
+interface IOncokbProps {
+    oncokb: IndicatorQueryResp | undefined;
+}
+
+export enum ONCOGENICITY {
+    ONCOGENIC = 'Oncogenic',
+    LIKELY_ONCOGENIC = 'Likely Oncogenic',
+    PREDICTED_ONCOGENIC = 'Predicted Oncogenic',
+    NEUTRAL = 'Neutral',
+    LIKELY_NEUTRAL = 'Likely Neutral',
+    INCONCLUSIVE = 'Inconclusive',
+    VUS = 'vus',
+    UNKNOWN = 'Unknown',
+}
+
+export const ONCOGENICITY_CLASS_NAMES: { [oncogenic: string]: string } = {
+    [ONCOGENICITY.ONCOGENIC]: 'oncogenic',
+    [ONCOGENICITY.LIKELY_ONCOGENIC]: 'oncogenic',
+    [ONCOGENICITY.PREDICTED_ONCOGENIC]: 'oncogenic',
+    [ONCOGENICITY.NEUTRAL]: 'neutral',
+    [ONCOGENICITY.LIKELY_NEUTRAL]: 'neutral',
+    [ONCOGENICITY.INCONCLUSIVE]: 'inconclusive',
+    [ONCOGENICITY.VUS]: 'vus',
+    [ONCOGENICITY.UNKNOWN]: 'unknown',
+};
+
+export enum MUTATION_EFFECT {
+    GAIN_OF_FUNCTION = 'Gain-of-function',
+    LIKELY_GAIN_OF_FUNCTION = 'Likely Gain-of-function',
+    LOSS_OF_FUNCTION = 'Loss-of-function',
+    LIKELY_LOSS_OF_FUNCTION = 'Likely Loss-of-function',
+    SWITCH_OF_FUNCTION = 'Switch-of-function',
+    LIKELY_SWITCH_OF_FUNCTION = 'Likely Switch-of-function',
+    NEUTRAL = 'Neutral',
+    LIKELY_NEUTRAL = 'Likely Neutral',
+    INCONCLUSIVE = 'Inconclusive',
+    UNKNOWN = 'Unknown',
+}
+
+export const MUTATION_EFFECT_CLASS_NAMES: {
+    [mutationEffect: string]: string;
+} = {
+    [MUTATION_EFFECT.GAIN_OF_FUNCTION]: 'gain',
+    [MUTATION_EFFECT.LIKELY_GAIN_OF_FUNCTION]: 'gain',
+    [MUTATION_EFFECT.LOSS_OF_FUNCTION]: 'loss',
+    [MUTATION_EFFECT.LIKELY_LOSS_OF_FUNCTION]: 'loss',
+    [MUTATION_EFFECT.SWITCH_OF_FUNCTION]: 'switch',
+    [MUTATION_EFFECT.LIKELY_SWITCH_OF_FUNCTION]: 'switch',
+    [MUTATION_EFFECT.NEUTRAL]: 'neutral',
+    [MUTATION_EFFECT.LIKELY_NEUTRAL]: 'neutral',
+    [MUTATION_EFFECT.INCONCLUSIVE]: 'inconclusive',
+    [MUTATION_EFFECT.UNKNOWN]: 'unknown',
+};
+
+@observer
+class Oncokb extends React.Component<IOncokbProps> {
+    static ONCOKB_URL: string = 'https://oncokb.org/';
+
+    public oncogenicity(oncokb: IndicatorQueryResp) {
+        // TODO do we need to check the ""
```

Comment:
Is this `TODO` still valid or does it refer to `oncokb.oncogenic !== ''`?

Reply:
@onursumer TODO here is to remind me to double-check if the value could be "" and to see do we need the `oncokb.oncogenic !== ''`. The answer is it's possible to be "", so I should probably remove the TODO now

Reply:
@onursumer done